src/sorcha/modules/PPCalculateApparentMagnitudeInFilter.py

```diff
@@ -46,14 +53,14 @@ def PPCalculateApparentMagnitudeInFilter(
 
     H_col = "H_filter"
 
-    # first, get H, r, delta and alpha as ndarrays
-    # r, delta and alpha are converted to au from kilometres
-    r = padain["AstRange(km)"].values / 1.495978707e8
+    # first, get H, rho, delta and alpha as ndarrays
+    # delta, rho and alpha are converted to au from kilometres
+    delta = padain["AstRange(km)"].values / 1.495978707e8
```

Comment:
Since you're modifying this anyway, can we get rid of these hardcoded numbers?

Reply:
Absolutely. I was going to do it in a separate PR, but I can do it now no problem.

Reply:
Might as well do it in one go.

Reply:
Done.